docs/source/x_ext.rst

```diff
@@ -1287,19 +1299,19 @@ Handshake dependencies
 In order to avoid system level deadlock both the |processor| and the |coprocessor| shall obey the following rules:
 
 * The ``valid`` signal of a transaction shall not be dependent on the corresponding ``ready`` signal.
-* Transactions related to an earlier part of the instruction flow shall not depend on transactions with the same ``hartid`` and ``id`` related to a later part of the instruction flow. The instruction flow is defined from earlier to later as follows:
+* The only allowed dependencies between interfaces for transactions with the same ``hartid`` and ``id``are:
 
-  * compressed transaction
-  * issue transaction
-  * register transaction
-  * commit transaction
+  * ``issue_req`` may depend on ``compressed_resp``
+  * ``register`` may depend on ``compressed_resp``, ``issue_resp``, and ``commit``
```

Comment:
Why the allowed dependence on commit?

Reply:
This gets weird in case issue and register are coupled. In that case you allow register to depend on commit and you also allow commit to depend on issue (which is coupled to register).